builtin/game/chatcommands.lua

```diff
@@ -530,11 +530,16 @@ core.register_chatcommand("giveme", {
 })
 
 core.register_chatcommand("spawnentity", {
-	params = "<EntityName>",
-	description = "Spawn entity at your position",
+	params = "<EntityName> [<X>,<Y>,<Z>]",
+	description = "Spawn entity at given (or your) position",
 	privs = {give=true, interact=true},
 	func = function(name, param)
-		local entityname = string.match(param, "(.+)$")
+		local entityname = string.match(param, "^([^ ]+)$")
+		local p
+		if not entityname then
```

Comment:
Regex was actually taken from teleport formula. Note that this allow separating coordinates by space as well as by comma.